AirCasting/Notes/NotesHandler.swift

```diff
@@ -89,7 +89,7 @@ class NotesHandlerDefault: NSObject, NotesHandler, NSFetchedResultsControllerDel
             do {
                 try storage.updateNote(note, newText: newText, for: sessionUUID)
                 fetchSession(storage: storage) { session in
-                    self.sessionUpdateService.updateSession(session: session) { result in
+                    self.sessionUpdateService.updateSession(session: session) { [self] result in
```

Comment:
note that this doesn't change anything in practice. The block still captures `self` strongly